stanford_news.info.yml

```diff
@@ -1,51 +1,47 @@
 name: 'Stanford News'
 type: module
 description: 'Stanford news items and views.'
-core_version_requirement: ^8.8 || ^9
+core_version_requirement: '^8.8 || ^9'
 package: Stanford
-version: 8.x-2.0
+version: 8.x-2.1
 dependencies:
-  - allowed_formats:allowed_formats
-  - auto_entitylabel:auto_entitylabel
-  - components:components
-  - drual:field
-  - drupal:breakpoint
-  - drupal:datetime
-  - drupal:file
-  - drupal:image
-  - drupal:layout_builder
-  - drupal:link
-  - drupal:menu_ui
-  - drupal:node
-  - drupal:path
-  - drupal:taxonomy
-  - drupal:text
-  - drupal:user
-  - entity_reference_revisions:entity_reference_revisions
-  - field_formatter_class:field_formatter_class
-  - field_group:field_group
-  - jumpstart_ui:jumpstart_ui
-  - layout_builder_restrictions:layout_builder_restrictions
-  - menu_block:menu_block
-  - page_manager:page_manager
-  - page_manager:page_manager_ui
-  - paragraphs:paragraphs
-  - pathauto:pathauto
-  - rabbit_hole:rabbit_hole
-  - rabbit_home:rh_node
-  - stanford_media:stanford_media
-  - stanford_text_editor:stanford_text_editor
-  - taxonomy_menu:taxonomy_menu
-  - ui_patterns:ui_patterns
-  - ui_patterns:ui_patterns_layouts
-  - ui_patterns:ui_patterns_library
-  - ui_patterns:ui_patterns_views
-  - viewfield:viewfield
-  - views_block_filter_block:views_block_filter_block
-  - views_infinite_scroll:views_infinite_scroll
-  - views_taxonomy_term_name_depth:views_taxonomy_term_name_depth
-
+    - 'allowed_formats:allowed_formats'
+    - 'auto_entitylabel:auto_entitylabel'
+    - 'components:components'
+    - 'drual:field'
+    - 'drupal:breakpoint'
+    - 'drupal:datetime'
+    - 'drupal:file'
+    - 'drupal:image'
+    - 'drupal:layout_builder'
+    - 'drupal:link'
+    - 'drupal:menu_ui'
+    - 'drupal:node'
+    - 'drupal:path'
+    - 'drupal:taxonomy'
+    - 'drupal:text'
+    - 'drupal:user'
+    - 'entity_reference_revisions:entity_reference_revisions'
+    - 'field_formatter_class:field_formatter_class'
+    - 'field_group:field_group'
+    - 'jumpstart_ui:jumpstart_ui'
+    - 'layout_builder_restrictions:layout_builder_restrictions'
+    - 'menu_block:menu_block'
+    - 'page_manager:page_manager'
+    - 'page_manager:page_manager_ui'
+    - 'paragraphs:paragraphs'
+    - 'pathauto:pathauto'
+    - 'rabbit_hole:rabbit_hole'
+    - 'rabbit_home:rh_node'
+    - 'stanford_media:stanford_media'
+    - 'stanford_text_editor:stanford_text_editor'
+    - 'taxonomy_menu:taxonomy_menu'
+    - 'ui_patterns:ui_patterns'
+    - 'ui_patterns:ui_patterns_layouts'
+    - 'ui_patterns:ui_patterns_library'
+    - 'ui_patterns:ui_patterns_views'
+    - 'views_block_filter_block:views_block_filter_block'
+    - 'views_infinite_scroll:views_infinite_scroll'
+    - 'views_taxonomy_term_name_depth:views_taxonomy_term_name_depth'
 component-libraries:
-  news:
-    paths:
-      - templates/components
+    news: { paths: [templates/components] }
```

Comment:
Why the change to the format?

Reply:
That's not a change I made manually; the php YAML library does that for some reason.  It's not necessary.

Reply:
Ah, I figured it had to do with yaml parsing and outputting.

Reply:
fixed it.